lighthouse-core/audits/byte-efficiency/byte-efficiency-audit.js

```diff
@@ -105,9 +105,12 @@ class UnusedBytes extends Audit {
    * @param {Array<LH.Audit.ByteEfficiencyResult>} results The array of byte savings results per resource
    * @param {Node} graph
    * @param {Simulator} simulator
+   * @param {{includeLoad?: boolean}=} options
    * @return {number}
    */
-  static computeWasteWithTTIGraph(results, graph, simulator) {
+  static computeWasteWithTTIGraph(results, graph, simulator, options) {
```

Comment:
add docstring for `includeLoad` option :)

Reply:
done